src/index.html

```diff
@@ -5,9 +5,38 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img
+         src="images/imac.jpeg"
+         alt="APPLE A1419 iMac 27"
+         class="card__image"
+      >
+      <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__review-container">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <span class="card__review-count">Reviews: 5</span>
+      </div>
+      <div class="card__price-container">
+        <span class="card__price-label">Price:</span>
+        <span class="card__price-value">$2,199</span>
+      </div>
+      <a
+        data-qa="hover"
+        href="#buy"
+        class="card__button">buy</a>
```

Comment:
```suggestion
      <a
        data-qa="hover"
        href="#buy"
        class="card__button"
      >
        buy
      </a>
```

Check please this point
![Снимок экрана 2023-09-07 в 14 10 26](https://github.com/mate-academy/layout_product-cards/assets/75898310/bb7a4342-5b84-43c6-b56a-974dbbe4ec25)